helm-charts/README.md

```diff
@@ -2,26 +2,29 @@
 
 Welcome to Infisical Helm Charts repository! Find instructions below to setup and install our charts.
 
+## Installation
+
 ```sh
 # Add the Infisical repository
 helm repo add infisical 'https://dl.cloudsmith.io/public/infisical/helm-charts/helm/charts/' && helm repo update
 
-# Install Infisical
+# Install Infisical (default values)
 helm upgrade --install --atomic \
```

Comment:
I found `--atomic` to be a bit strange since it also adds --wait. With this, it kind of looks like the installation is  just frozen 🤔